app/helpers/application_helper/toolbar/diagnostics_server_center.rb

```diff
@@ -40,9 +40,9 @@ class ApplicationHelper::Toolbar::DiagnosticsServerCenter < ApplicationHelper::T
       :klass => ApplicationHelper::Button::DiagnosticsEvmLogs),
     button(
       :refresh_production_log,
-      'fa fa-repeat fa-lg',
+      'fa fa-refresh fa-lg',
       proc do
-        _('Reload the %{log_type} Log Display') % {:log_type => _(@sb[:rails_log])}
+        _('Refresh this page') % {:log_type => _(@sb[:rails_log])}
```

Comment:
No need to pass in the `:log_type` value since the text no longer contains the substitution.